utility/src/main/java/org/oppia/android/util/profile/BUILD.bazel

```diff
@@ -14,3 +14,11 @@ kt_android_library(
         "//third_party:javax_inject_javax_inject",
     ],
 )
+
+kt_android_library(
+    name = "name_validator",
+    srcs = [
+        "NameValidator.kt",
+    ],
+    visibility = ["//domain:__subpackages__","//utility:__subpackages__"],
```

Comment:
I suggest just making this API visibility (i.e. ``//:oppia_api_visibility``) since we don't specify more granular visibility unless it's a relative sub-package, test-only, or a module.